QuizApp/QuizApp/UI/Leaderboard/LeaderboardViewModel.swift

```diff
@@ -0,0 +1,32 @@
+import Combine
+
+class LeaderboardViewModel {
+
+    private let coordinator: CoordinatorProtocol
+    private let quizUseCase: QuizUseCaseProtocol
+
+    private var quizId: Int
+
+    @Published var leaderboardList: [QuizLeaderboard] = []
+
+    init(coordinator: CoordinatorProtocol, quizUseCase: QuizUseCaseProtocol, quizId: Int) {
```

Comment:
Since this VM is using just one UseCase, you can name the argument and field just useCase.